Runtime/SpriteSwitchMonoBehaviour.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.MaterialSwitch
+{
+    [ExecuteAlways]
```

Comment:
Does it make sense for users to add this by themselves ? 
If not, we can use `[AddComponentMenu("")]` to prevent this.

Also, we should probably use a more appropriate name.
`SpriteSwitchEventInvoker`, perhaps